examples/kubernetes/create_secrets.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Sure, we could do that. The intent was just to have a way of creating secrets without creating everything else along with it.